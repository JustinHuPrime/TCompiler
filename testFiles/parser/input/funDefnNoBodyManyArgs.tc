module foo;

int bar(int arg1, void * arg2, int arg3) {}