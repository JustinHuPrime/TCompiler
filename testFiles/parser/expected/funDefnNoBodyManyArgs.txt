testFiles/parser/funDefnNoBodyManyArgs.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/funDefnNoBodyManyArgs.tc, 3, 1, int(int, void *, int)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, int), ID(3, 5, bar, REFERENCES(testFiles/parser/funDefnNoBodyManyArgs.tc, 3, 1)), KEYWORDTYPE(3, 9, int), MODIFIEDTYPE(3, 19, POINTER, KEYWORDTYPE(3, 19, void)), KEYWORDTYPE(3, 32, int), ID(3, 13, arg1, REFERENCES()), ID(3, 26, arg2, REFERENCES()), ID(3, 36, arg3, REFERENCES()), STAB(ENTRY(arg1, VARIABLE(testFiles/parser/funDefnNoBodyManyArgs.tc, 3, 9, int)), ENTRY(arg3, VARIABLE(testFiles/parser/funDefnNoBodyManyArgs.tc, 3, 32, int)), ENTRY(arg2, VARIABLE(testFiles/parser/funDefnNoBodyManyArgs.tc, 3, 19, void *))), COMPOUNDSTMT(3, 42, STAB())))
